### ISSUE TITLE
A possible leak when in "standby" mode

### ISSUE THREAD

**Issue by nikita-volkov** [scotty-web/scotty#151]
I have a Scotty application. When I start it, the `top` command shows that it consumes ~ 1% of CPU. If I leave it running and say for a day nobody makes a single request to it, the CPU consumption grows to 25%, and keeps growing if I keep the app running. So it evidences a leak. However, as soon as I make a single request to the app, the CPU consumption drops to initial levels.

**Comment by xich** [comment]
Interesting. Have you tried doing this with a plain WAI application? That would tell us if the leak is in Scotty or WAI/warp.

**Comment by nikita-volkov** [comment]
No, I haven't. I've made a basic analysis of my app and it seems like Scotty is the cause. Possibly, some time later I'll do a more thorough analysis of my app to get to the bottom of this, but I can't promise. 

Any way, I think it's best to at least keep this issue open in case somebody else bumps into the same problem, thus confirming that Scotty is the cause.

**Comment by chessai** [comment]
@nikita-volkov this is a longshot, but do you happen to have a copy of the code or a minimal reproduction available?

**Comment by nikita-volkov** [comment]
@chessai No unfortunately
